props/saber_fett263_buttons.h

```diff
@@ -5593,7 +5598,37 @@ SaberFett263Buttons() : PropBase() {}
         clash_type_ = CLASH_BATTLE_MODE;
         return true;
 
+#ifdef FETT263_USE_BC_MELT_STAB
+      case EVENTID(BUTTON_NONE, EVENT_STAB, MODE_ON):
```

Comment:
Please move all the parts that are the the same (top and bottom) to outside of the #ifdef

Reply:
Updated